client/components/App.jsx

```diff
@@ -1,28 +1,46 @@
 import React from 'react'
-import {BrowserRouter as Router, Route} from 'react-router-dom'
+import {Route, withRouter} from 'react-router-dom'
 
 import '../styling/main.css'
 
 import Home from './Home'
 import Header from './Header'
 import SignIn from './SignIn'
-import Register from './Register'
 import Profile from './Profile'
+import Register from './Register'
```

Comment:
If you moved this line down for aesthetic reasons, you are _definitely_ strumming my heart strings. Thank you :heart: